reference/employer/openapi.yaml

```diff
@@ -373,7 +373,7 @@ paths:
                               nullable: true
                             ssn:
                               type: string
-                              description: 'Social Security number of the individual. This field is only available with the `ssn` scope enabled and the `options: { include: [''ssn''] }` param set in the body.'
+                              description: "Social Security number of the individual. This field is only available with the `ssn` scope enabled and the `options: { include: ['ssn'] }` param set in the body."
```

Comment:
I think the quotes are necessary because of the presence of `:` in the sentence. I also prefer the double quotes as opposed to doubling up the single quotes here: `{ include: [''ssn''] }`